include/ceed/sycl.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2017-2022, Lawrence Livermore National Security, LLC and other CEED contributors.
+// All Rights Reserved. See the top-level LICENSE and NOTICE files for details.
+//
+// SPDX-License-Identifier: BSD-2-Clause
+//
+// This file is part of CEED:  http://github.com/ceed
+
+#ifndef _ceed_sycl_h
+#define _ceed_sycl_h
+
+#include <ceed/ceed.h>
+
+CEED_EXTERN int CeedSetStream_Sycl(Ceed ceed, void* handle);
+
+#endif
```

Comment:
I don't think we need this anymore? We moved to a generic `CeedSetStream`, so this doesn't need to be public facing anymore.

Reply:
@kris-rowe - It looks the registering this backend function is done inside the sycl-ref init function - https://github.com/argonne-lcf/libCEED/blob/fbf6e8e0622db949ec8e1942d099926308ba6a9d/backends/sycl-ref/ceed-sycl-ref.sycl.cpp#L41
Do you think this will take care of all backends or does it have to be included for each specific backend? Or maybe in the common sycl init function? Like here -  https://github.com/argonne-lcf/libCEED/blob/fbf6e8e0622db949ec8e1942d099926308ba6a9d/backends/sycl/ceed-sycl-common.sycl.cpp#L74

Reply:
Registering in sycl-ref init should be sufficient (e.g., like `GetPreferredMemType`), since the sycl-shared and sycl-gen backends will use the same function through delegation.